src-relution/styles/src/_mw-file-upload.scss

```diff
@@ -53,7 +53,6 @@
   .upload-info {
     padding-bottom: 10px;
     position: relative;
-    font-size: 11px;
```

Comment:
The font size was set to 11px so the upload pgrogressbar has the same height as the cancel button